tests/unit/utils/scheduler/test_schedule.py

```diff
@@ -459,3 +457,39 @@ def test_handle_func_check_data(self):
 
             self.assertIn("kwargs", data)
             self.assertNotIn("__pub_fun_args", data["kwargs"])
+
+    @skipIf(not salt.utils.platform.is_windows(), "Skip on Non-Windows systems")
+    def test_handle_func_check_dicts(self):
+        """
+        Tests that utils, functions, and returners dicts are not
+        empty after handle_func has run on Windows.
+        """
+        self.schedule.standalone = True
```

Comment:
This will affect all tests running after this one.
If they should all run in `standalone` mode then this should be set when instantiating `self.scheduler`, otherwise, you should use `mock.patch` here...